cities/routers.py

```diff
@@ -0,0 +1,45 @@
+from fastapi import APIRouter, Depends, HTTPException
+from sqlalchemy.ext.asyncio import AsyncSession
+from cities import schemas, crud
+from dependencies import get_db
+
+
+router = APIRouter()
+
+
+@router.post("/cities/", response_model=schemas.CityResponse)
+async def create_city(
+    city: schemas.CityCreate, db: AsyncSession = Depends(get_db)
+):
+    return await crud.create_city(db=db, city=city)
+
+
```

Comment:
The return type hint `list[schemas.CityResponse]` requires Python 3.9 or later. If you're using an earlier version, consider using `List[schemas.CityResponse]` from the `typing` module.